app/models/user.rb

```diff
@@ -1,18 +1,21 @@
 # frozen_string_literal: true
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+    :recoverable, :rememberable, :validatable,
+    :confirmable, :lockable, :trackable,
+    :argon2, argon2_options: { profile: Rails.env.test? ? :unsafe_cheapest : :rfc_9106_low_memory }
```

Comment:
Do we actually want trackable? the rest looks good tho after I looked them up

Reply:
I'm not entirely sure. I thought it could be useful in case someone's account does get compromised - they might want to know details about when/how it happened, and it would be hard for us to look up in the logs - but we don't need it. Do you think it would be best to remove it?

Reply:
That makes sense! I mostly wanted to check :bug: that it'd been considered and there was a reason, rather than it just being in by default?

Reply:
That makes sense! I tried to come up with a reason for each of the ones I included here, but trackable and lockable are the ones I had weakest reasoning for/against, since we don't have this functionality right now. It's not enabled by default (if I recall correctly), so I actively chose to enable it here just to have more information available, but only with weak reasoning =) If you have thoughts either way, it would be great to hear them!

Reply:
(discussing in #devs, will put consensus in TODOs)